plugins/woocommerce-admin/client/two-column-tasks/sectioned-task-list.tsx

```diff
@@ -54,6 +54,10 @@ export const SectionedTaskList: React.FC< TaskListProps > = ( {
 	const [ openPanel, setOpenPanel ] = useState< string | null >(
 		sections?.find( ( section ) => ! section.isComplete )?.id || null
 	);
+	const [
+		isCompletedCardVisible,
+		setIsCompletedCardVisible,
```

Comment:
Took me a minute to grok what was happening, but I think a more intentional name might be `shouldKeepTasks`, what do you think?

Reply:
Good idea! I renamed the const in commit [f621b4e](https://github.com/woocommerce/woocommerce/pull/33238/commits/f621b4e22e3031e2ae9e4ed5eb4f6d2b61d5c82b)